fiftyone_devicedetection_cloud/tests/test_properties.py

```diff
@@ -114,7 +113,10 @@ def test_value_types(self):
             expected_type = propertymeta["type"]
             if(property in dd_property_dict and property not in exclude_properties):
                 dd_property_value = dd_property_dict[property]
-                value = dd_property_value.value()
-                self.assertIsNotNone("Property: " + property +" is not present in the results.", dd_property_value)
-                self.assertTrue("Expected type for " + property + " is " + expected_type + 
-                " but actual type is " + get_value_type(value), is_same_type(value, expected_type))
+                try:
```

Comment:
also this `try` will catch all `AssertError` exceptions and this test will always succeed, see example: 
<img src="https://github.com/51Degrees/device-detection-python/assets/341967/fdf17347-f511-413e-a4d3-a94f19104d02" width="400px" height="400px" />

we need to wrap only value extraction into try/catch, and then check if the property name is "javascriptgethighentropyhints" (or which exact property did not have a value and gave a null reason) and pass checks for that property explicitly.